plugins/server/dev.hayden/khealth/versions.ktor.yaml

```
@@ -0,0 +1,2 @@
+"[2.0,)": dev.hayden:khealth:2.1.+
```

Looks like this open range is why the generator is selecting the 2.1.1 version.

```suggestion
"[2.0,3.0)": dev.hayden:khealth:2.1.+
```

You'll need to rename the folder to "2.0,3.0" as well.